src/countervalues/index.js

```diff
@@ -598,7 +598,9 @@ function createCounterValues<State>({
       if (prevProps.pairsKey !== this.props.pairsKey) {
         if (log)
           log(
-            `pairsKey changed:\n    ${prevProps.pairsKey}\n => ${this.props.pairsKey}`
+            `pairsKey changed:\n    ${prevProps.pairsKey}\n => ${
+              this.props.pairsKey
```

Comment:
I suspect you have diff prettier version? can you check if `yarn prettier` don't re-set this to original? @juan-cortes